app/terms-condition/page.tsx

```diff
@@ -11,52 +13,133 @@ export default function TermsCondition(){
       margin: '0 auto',
       padding: '25px'
     }}>
-     <center> <h1> <b> Terms & Conditions </b> </h1> </center> 
-     <center> <p> Our friendly team would love to hear from you. </p> </center>
-
-      <h1> Terms and Conditions </h1>
-      <h2> Effective Date: <b> May 8, 2023 </b></h2>
-      <p> These Terms and Conditions (“Terms”) govern your use of the SLC website and the services provided by SLC (“we,” “us,” or “our”). By accessing our website or using our services, you agree to be bound by these Terms. If you do not agree with these Terms, please refrain from using our website and services.</p>
       
-      <h1> 1. Acceptance of Terms </h1>
-      <h3> 1.1 Agreement to Terms: </h3>
-      <p> By accessing our website or using our services, you acknowledge that you have read, understood, and agree to be bound by these Terms. If you do not agree to these Terms, you may not use our website or services. </p>
+     <center> <Typography variant="h4"> 
+          <b> { pageDetails.title } </b> 
+          </Typography> 
+     </center> 
+     
+     <center> 
```

Comment:
Avoid using HTML elements. Instead use MUI alternatives. You can use the textAlign property directly on the Typography component to achieve center alignment as below. 

<Typography variant="h4" align="center">